### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by AskAkshith** [uxbox/uxbox#1547] [Ideas]
Hello penpot Team, First of all congratulations and thank you for developing designing platform like penpot. I have been using it for my designing(Both Graphic and UI designing). Coming to features in penpot one feature that i liked most is "assets", It categories all the components in our designs. After using it for my projects i also thought that in addition to it you can make improvement in it by extending to allowing us to get designs directly from internet(like, if we need any logo like search, arrows , mail icon, facebook icon, etc then by searching it in assets search bar we should be able to find them and can use them by dragging them to artboard(Canva has this feature)). Please look into it and add this feature in assets so that we can use logos,icons,images etc directly from internet by searching them. This feature will help a lot of designers and make there work easy and more professional.

**Comment by hirunatan** [comment]
Hello. You are true, this would be a very powerful feature. But also a complex one, because we need to cope with content licenses, file formats and access APIs. We have in our backlog some stories related to this, like this one https://tree.taiga.io/project/penpot/us/2236

This is one of the important and big functions to be implemented in Penpot, but it needs to be prioritized against other equally big.

Also you can check the public libraries and templates, because they have collections of assets ready to use: https://penpot.app/libraries-templates.html

**Comment by AskAkshith** [comment]
Unsplash's integration we will be a great help for designers to import images from online, but also add feature to upload icons and also import SVG files from local system. Will can't move our designs which are in SVG format to penpot since penpot is only accepting .penpot file formats.
A feature to import Figma(.fig), Adobe XD(.XD), Sketch(.sketch), SVG(.svg) will be a great help for designers and also gain turn designers who are using design softwares like Figma, Adobe XD, sketch, Adobe Illustrator to penpot users.
